xbmc/dialogs/GUIDialogSeekBar.cpp

```diff
@@ -67,7 +67,7 @@ void CGUIDialogSeekBar::FrameMove()
     return;
   }
 
-  unsigned int percent((!CSeekHandler::GetInstance().InProgress() && g_infoManager.GetTotalPlayTime())
+  unsigned int percent((!g_application.m_pPlayer->GetSeekHandler().InProgress() && g_infoManager.GetTotalPlayTime())
```

Comment:
That notation is a bit "odd" or let's say I am not used to of this multiple bool Int mix. Why not: unsigned int percentage = InProgress() ? GetTotalPlaytime : 0;

Reply:
Logic is untouched by me, should I change it to what you suggested in a separate commit?

Reply:
after a second look it's a ternary operator.

```c++
unsigned int percent((!g_application.m_pPlayer->GetSeekHandler().InProgress() && g_infoManager.GetTotalPlayTime())
  ? lrintf(g_application.GetPercentage())
  : (unsigned int)g_infoManager.GetSeekPercent());
```
same like

```c++
unsigned int percent = !g_application.m_pPlayer->GetSeekHandler().InProgress() && g_infoManager.GetTotalPlayTime()
  ? lrintf(g_application.GetPercentage())
  : static_cast<unsigned int>(g_infoManager.GetSeekPercent());
```

Reply:
Ah! Thanks much. Now it makes sense - no need to change anything, besides the global :-)

Reply:
Np, not as easy to read I agree, because of the missing `=` and the surrounding braces ;)